app/policies/portfolio_item_policy.rb

```diff
@@ -8,22 +8,34 @@ def create?
   end
 
   def update?
-    rbac_access.update_access_check
+    rbac_access.resource_check('update', @record.portfolio_id, Portfolio)
   end
 
   def destroy?
-    rbac_access.destroy_access_check
+    rbac_access.resource_check('update', @record.portfolio_id, Portfolio)
```

Comment:
@eclarizio Isn't this a Catalog_admin? check because only Administrators can delete a Portfolio Item.

Reply:
@eclarizio Looks good verified the doc with @gmcculloug